docs/model_training.ipynb

```diff
@@ -401,8 +401,11 @@
   }
  ],
  "metadata": {
+  "interpreter": {
+   "hash": "6919b47cc8f8d676c7fed1b967e6a734dcd85e514e7a620e6ce1fdd963adcf4d"
+  },
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3.11.9 ('routee_compt')",
```

Comment:
Can you revert all the changes made to this file? We don't want to update the `kernelspec` since the kernel `routee_compt` only exists on your computer